magit-gh-comments.el

```diff
@@ -445,32 +471,81 @@ which they came, add them to current magit-diff buffer."
+        (magit-insert-section (summary)
+          (magit-insert-heading "FIXME - PR TITLE HERE"))
+        (let ((reviews (magit-gh--list-reviews pr))
+              (diff-body (magit-gh--fetch-diff-from-github pr)))
+          (dolist (review reviews)
+            (magit-insert-section (review review)
+              (magit-insert-heading (format "Review by %s"
+                                            (alist-get :author review)))
+              (when-let ((body (alist-get :body review)))
+                (insert body "\n"))
+              (dolist (comment (alist-get :comments review))
+                (magit-insert-section (comment comment)
+                  (insert (magit-gh--propertize-comment-ctx diff-body
+                                                            (magit-gh-pr-diff-range pr)
+                                                            comment))
+                  (insert "\n")
+                  (insert (alist-get :body comment))
+                  (insert "\n")
+                  (insert (format "- %s" (alist-get :author comment)))
+                  (insert "\n")))))))
+          (goto-char (point-min)))
+    (get-buffer buf-name)))
+
+(defun magit-gh--propertize-comment-ctx (diff-body diff-range comment)
+  (let* ((comment-ctx (magit-gh--comment-ctx
+                       diff-body
+                       (alist-get :position comment)
+                       (alist-get :path comment)))
+         (ctx-lines (split-string comment-ctx "\n"))
+         (i 0)
+         (in-hunk t)
+         result)
+    (dolist (line (reverse ctx-lines)
+                  (string-join result "\n"))
+      (setq in-hunk (and in-hunk
+                         (not (string-match-p magit-gh--hunk-header-re line))))
+      (setq result
+            (cons (if in-hunk
+                      (propertize line
+                                  'keymap
+                                  (let ((keymap (make-sparse-keymap)))
+                                    (define-key keymap (kbd "<return>")
+                                      (let ((gh-pos (- (alist-get :position comment) i)))
+                                        (lambda ()
+                                          (interactive)
+                                          (let ((magit-pos (magit-gh--diff-pos/gh->magit
+                                                            (alist-get :path comment)
+                                                            gh-pos
+                                                            diff-body)))
+                                            (magit-diff diff-range)
+                                            (magit-gh--visit-diff-pos (alist-get :path comment)
+                                                                      magit-pos)))))
+                                    keymap))
+                      line)
+                  result))
+      (setq i (1+ i)))))
+
+
+(defun magit-gh--jump-to-diff-for-comment ()
```

Comment:
This doesn't seem to do what it claims, and I don't see it used anywhere. Delete?